netdisco/lms.py

```diff
@@ -51,7 +51,7 @@ def update(self):
                         # And XXXX is the web interface port (ie 9000)
                         port = None
                         # https://github.com/python/typeshed/pull/2696
-                        if data.startswith(b'JSON', 1):  # type: ignore
```

Comment:
This was failing lots of peoples builds the past couple of weeks:

e.g.
https://travis-ci.org/home-assistant/netdisco/jobs/500069764